qiita_pet/handlers/artifact_handlers/base_handlers.py

```diff
@@ -8,6 +8,7 @@
 
 from os.path import basename, relpath
 from json import dumps
+from humanize import naturalsize
```

Comment:
Are you sure it's all right to use humanize? I looks like it last had a release just before 2015:

https://pypi.org/project/humanize/

Reply:
Yup, we have been using it for a while; note that it has been part of the setup.py for a while ...